modules/_labs/pagination/react/stories/stories.tsx

```diff
@@ -0,0 +1,133 @@
+/// <reference path="../../../../../typings.d.ts" />
+import styled from '@emotion/styled';
+import {boolean, number, text} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+import withReadme from 'storybook-readme/with-readme';
+import {ComponentStatesTable} from '../../../../../utils/storybook';
+
+import {Pagination} from '@workday/canvas-kit-labs-react-pagination';
+import README from '../README.md';
+
+const Wrapper = styled('div')({
+  display: 'block',
+  textAlign: 'center',
+});
+
+const getAriaLabels = () => ({
+  paginationContainerAriaLabel: text('paginationContainerAriaLabel', 'Pagination'),
+  previousPageAriaLabel: text('previousPageAriaLabel', 'Previous Page'),
+  nextPageAriaLabel: text('nextPageAriaLabel', 'Next Page'),
+});
+
+storiesOf('Labs|Pagination/React', module)
+  .addParameters({
+    component: Pagination,
+  })
+  .addDecorator(withReadme(README))
+  .add('Default', () => {
+    const [currentPage, setCurrentPage] = React.useState(1);
+
+    return (
+      <Wrapper>
+        <h4>
+          Current Page: <span data-testid="pageNumber">{currentPage}</span>
+        </h4>
+        <Pagination
+          total={number('total', 50) || 50}
+          pageSize={number('pageSize', 10) || 10}
+          showLabel={boolean('showLabel', false)}
+          showGoTo={boolean('showGoTo', false)}
+          goToLabel={text('goToLabel', 'Go To')}
+          currentPage={currentPage}
+          onPageChange={p => setCurrentPage(p)}
+          {...getAriaLabels()}
+        />
+      </Wrapper>
+    );
```

Comment:
These stories no longer handle auto-resize by the browser. Perhaps we should use `react-use` to add functionality back? I think `width` should be provided for greater flexibility, but our stories should show how to pass it in.